code/game/objects/machinery/mortar.dm

```diff
@@ -373,6 +395,16 @@
 		return
 	return ..()
 
+// Artillery cameras. Together with the artillery impact hud tablet, shows a live feed of imapcts.
+
+/obj/machinery/camera/artillery
+	name = "artillery camera"
+	network = list("terragovartillery")
+	alpha = 0 //we shouldn't be able to see this!
+	internal_light = FALSE
+	c_tag = "impact camera"
+	resistance_flags = RESIST_ALL
+
```

Comment:
this seems like  a vary random place to define this shouldnt you put it above the mortar definition?